x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsCanMatchOnCoordinatorIntegTests.java

```diff
@@ -622,7 +627,22 @@ public void testSearchableSnapshotShardsThatHaveMatchingDataAreNotSkippedOnTheCo
 
         // All shards failed, since all shards are unassigned and the IndexMetadata min/max timestamp
         // is not available yet
-        expectThrows(SearchPhaseExecutionException.class, () -> client().search(request).actionGet());
+        try {
+            SearchResponse response = client().search(request).actionGet();
```

Comment:
Since `expectThrows` accepts a lambda you could have done it without the manual try/catch.

```java
expectThrows(SearchPhaseExecutionException.class, {
        var response = client().search(request).actionGet();
        log.info - message about response.
});
```